src/cmd/kubectl.rs

```diff
@@ -1017,6 +1017,88 @@ where
     )
 }
 
+/// kubectl_get_crash_looping_pods: gets crash looping pods.
+///
+/// Arguments
+///
+/// * `kubernetes_config`: kubernetes config file path.
+/// * `namespace`: namespace to look into, if None, will look into all namespaces.
+/// * `selector`: selector to look for, if None, will look for anything.
+/// * `restarted_min_count`: minimum restart counts to be considered as crash looping. If None, default is 10.
+/// * `envs`: environment variables to be passed to kubectl.
+pub fn kubectl_get_crash_looping_pods<P>(
+    kubernetes_config: P,
+    namespace: Option<&str>,
+    selector: Option<&str>,
+    restarted_min_count: Option<usize>,
+    envs: Vec<(&str, &str)>,
+) -> Result<Vec<KubernetesPod>, SimpleError>
+where
+    P: AsRef<Path>,
+{
+    let restarted_min = restarted_min_count.unwrap_or_else(|| 10usize);
+    let pods = kubectl_exec_get_pods(kubernetes_config, namespace, selector, envs)?;
+
+    Ok(pods
+        .items
+        .into_iter()
+        .filter(|e| {
```

Comment:
does e stand for pod ? If you yes, I'd suggest to switch to an explicit var

Reply:
Yep, ok !